controllers/sriovoperatorconfig_controller.go

```diff
@@ -95,16 +95,20 @@ func (r *SriovOperatorConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Result
 		return reconcile.Result{}, nil
 	}
 
-	if enableAdmissionController {
-		// Render and sync webhook objects
-		if err = r.syncWebhookObjs(defaultConfig); err != nil {
-			return reconcile.Result{}, err
-		}
+	// Render and sync webhook objects
+	if err = r.syncWebhookObjs(defaultConfig); err != nil {
```

Comment:
Would this fail in non-openshift environment when calling `r.deleteWebhookObject`?

Reply:
No, it would not. The function only handles k8s objects. Those are common in openshift and vanilla k8s.